src/python/dxpy/scripts/dx_build_app.py

```diff
@@ -194,7 +197,7 @@ def _find_readme(dirname):
     if 'name' in app_spec:
         if app_spec['name'] != app_spec['name'].lower():
             logger.warn('name "%s" should be all lowercase' % (app_spec['name'],))
-        if dirname != app_spec['name']:
+        if dirname != app_spec['name'] and not str(os.path.abspath(dxapp_json_filename)).startswith("/tmp") and not dirname.startswith("."):
```

Comment:
When I build a regular applet with "dx build ." but the current dir name != app name in dxapp.json this will not be printed any more, right? Some tests may fail.